modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -588,6 +597,10 @@ impl ServerModule for Wallet {
 
                     dbtx.remove_entry(&PegOutTxSignatureCI(txid)).await;
                     dbtx.remove_entry(&UnsignedTransactionKey(txid)).await;
+                    let propose_citem_tx = self.propose_citem.clone();
+                    dbtx.on_commit(move || {
+                        propose_citem_tx.notify_one();
```

Comment:
Why notify here?

I believe this will cause redundant block count and fee rate votes. The`PegOutTxSignatureCI` entry was removed, so no new information needs to be submitted.

Not a regression since this is the behavior without the changes in this PR.